views/productDetail.pug

```diff
@@ -0,0 +1,15 @@
+doctype html
+html
+  head
+    title Product Details
+    include stylesheets.pug
+  body
+    include ./partials/nav.pug
+    
+  block content
+    h1 Product Details
```

Comment:
You may want to provide for 'what if' a user inputs a url with a number or something the system can't handle. Just something to think about ....

Reply:
Hey @Rian501, thanks for the great suggestion! This is going to be handled by another teammate later on in the sprint. Let me know if you have any questions!